README.md

```diff
@@ -60,6 +46,33 @@ The full `Bucket` state is replicated which fits in less than 256 bytes.
 Together with its merge semantics, this makes a `Bucket` a **state based**
 *Convergent Replicated Data Type* (CvRDT) based on a [PN-Counter](https://en.wikipedia.org/wiki/Conflict-free_replicated_data_type#PN-Counter_(Positive-Negative_Counter)).
 
+### Clock synchronization
+
+While the sort of rate limiting supported by Patrol is time based (e.g 100 requests **per minute**),
+it **does not** depend on node clocks to be synchronized (e.g via `ntpd`).
+
+A `Bucket` stores the local time when it was created in a given node as well as a relative
+elapsed duration which represents how much the `Bucket` has advanced in time via successful `Takes`.
+Only the elapsed duration is replicated to other nodes and merged with CRDT G-Counter semantics;
+this value is global across the cluster, but is added to the local, per Bucket `Created` timestamp
+to calculate time deltas between successive `Take` operations and, hence, the number of tokens to
+refill over time.
+
+By keeping the `Created` timestamps local and using only relative time arithmetic, we avoid the
+need to synchronize clocks across the cluster.
+
+### Consistency, Availability, Partition-Tolerance (CAP)
+
+Under a network partition, nodes won't be able to actively replicate `Bucket` state to nodes on
+other sides of the partition. The effect of this is that a `Bucket`'s global rate limit
+will be multiplied by the number of sides in a partition and will lead to temporary policy
+violations until the partition heals; in other words, Patrol fails-open under netsplits.
```

Comment:
"Fails open" means it would _stop_ requests from flowing in a failure scenario. But I think you are "fails closed" because you _permit_ requests to flow in a failure scenario. Do I have it wrong?

Reply:
Thanks, I switched the terms around. I’ll fix it.

Reply:
Done in https://github.com/tsenart/patrol/commit/e46543c39333a390ad51fd26125e64891eb164e6